src/routes/UserRoute.tsx

```diff
@@ -1,6 +0,0 @@
-import React from 'react';
-import { Outlet } from 'react-router-dom';
-
-const UserRoute = () => <Outlet />;
```

Comment:
This will be brought back once we implement public user profiles, but there's not much of value in this file, so anything in here will be remade